### DISCUSSION TITLE
Welcome to GSD Discussions!

### DISCUSSION THREAD

**Discussion by dcaddick** [dcaddick/gsd_public#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DeanGross** [comment]
David, we have chatted online many times in the past, hopefully we will get to meet in person sometime. I'm based in Charlotte NC and like you, I focus on helping our clients improve their security posture and processes with the wide variety of Microsoft products.
